### ISSUE TITLE
Multiple scss syntax errors

### ISSUE THREAD

**Issue by muschottkey** [tatygrassini/landio-html#7]
Hi,

I'm getting multiple errors regarding syntax and import import directives being used inside control structures. Whereas I simply removed the imports inside @if and @include, the syntax errors regarding lists isn't going away.

Sample error messages include 

```
error scss/landio.scss (Line 64 of scss/bootstrap/_custom-variables.scss: Invalid CSS after "  0": expected ")", was ": (")
error scss/bootstrap/bootstrap-grid.scss (Line 19: Invalid CSS after "  xs": expected ")", was ": 0,")
error scss/bootstrap/bootstrap.scss (Line 64 of scss/bootstrap/_variables.scss: Invalid CSS after "  0": expected ")", was ": (")
```

After checking the files, the errors are coming from declarations as these 

```
$spacers: (
  0: (
    x:   0,
    y:   0
  ),
  1: (
    x:   $spacer-x,
    y:   $spacer-y
  ),
  2: (
    x:   ($spacer-x * 1.5),
    y:   ($spacer-y * 1.5)
  ),
  3: (
    x:   ($spacer-x * 3),
    y:   ($spacer-y * 3)
  )
) !default;
```

I'm using Sass 3.4.21 (Selective Steve). How can I resolve this without having to rewrite/redo your work?

**Comment by owenpt** [comment]
I guess (hope) you've got this sorted now. For anyone else who finds this, looking for a solution, check to see if your compiler is the problem.

I went to try and create a fix by editing the code, but I'm glad I thought to check my compiler first! I was using Scout, gave Koala a try and it compiled first time correctly. Happy days :).

**Comment by tatygrassini** [comment]
@muschottkey & @owenpt The main reason I created this template is to push Bootstrap 4 development forward and support for any Landio related bugs.

These errors are Bootstrap and Sass related and should be addressed on the [Bootstrap](https://github.com/twbs/bootstrap) or [Sass](https://github.com/sass/sass) repos.

I ran this repo through Scout myself (nobody in the world should be using such a lame app) and I got this error: `Import directives may not be used within control directives or mixins.` This is a Sass bug reported [here](https://github.com/sass/sass/issues/451).

Not sure how people compiles this template, but it's always a good idea to have the latest Sass version installed on your machine. I personally created this template using Codekit and I included a config file for it in the repo. And no, no Scss bugs with it.
